book/src/drivers.md

```diff
@@ -32,6 +32,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [CubeFS](https://github.com/cubefs/cubefs-csi) | `csi.cubefs.com` | v1.1.0 | A Container Storage Interface (CSI) Driver for CubeFS Storage  | Persistent | Read/Write Multiple Pods | Yes |
 [Cinder](https://github.com/kubernetes/cloud-provider-openstack/tree/master/pkg/csi/cinder) | `cinder.csi.openstack.org` | v0.3, v1.0, v1.1.0, v1.2.0, v1.3.0 | A Container Storage Interface (CSI) Driver for OpenStack Cinder | Persistent and Ephemeral | Depends on the storage backend used | Yes, if storage backend supports it | Raw Block, Snapshot, Expansion, Cloning, Topology
 [cloudscale.ch](https://github.com/cloudscale-ch/csi-cloudscale) | `csi.cloudscale.ch` | v1.0 | A Container Storage Interface (CSI) Driver for the [cloudscale.ch](https://www.cloudscale.ch/) IaaS platform | Persistent | Read/Write Single Pod | Yes |Snapshot
+[CTDI Block Storage](https://ctdi-storage.github.io/) | `csi.block.ctdi.com` | v1.0 | A Container Storage Interface (CSI) Driver for CTDI Distributed Block Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
```

Comment:
Can you please provide an English version of Deployment instructions on the github?  It is in Chinese now.
In the instruction steps, you mentioned a few yamls.  Where do user find those yamls?  Maybe I missed something here.
Also Column 3 is for supported CSI spec version, not CSI driver version.

Reply:
@xing-yang I have uploaded the English version of Deployment instructions and updated the CSI spec version